apps/api/src/app/workflows/workflow.module.ts

```diff
@@ -7,11 +7,13 @@ import { ChangeModule } from '../change/change.module';
 import { AuthModule } from '../auth/auth.module';
 import { IntegrationModule } from '../integrations/integrations.module';
 import { WorkflowController } from './workflow.controller';
+import { GetPreferences, UpsertPreferences } from '@novu/application-generic';
+import { PreferencesRepository } from '@novu/dal';
 
 @Module({
   imports: [SharedModule, MessageTemplateModule, ChangeModule, AuthModule, IntegrationModule],
   controllers: [NotificationTemplateController, WorkflowController],
-  providers: [...USE_CASES],
+  providers: [...USE_CASES, GetPreferences, UpsertPreferences, PreferencesRepository],
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Add to `USE_CASES` instead? If not, please comment in code why not‏